apps/extension/src/pages/formatting.tsx

```diff
@@ -0,0 +1,287 @@
+          DateFormat.yyyy_MM_dd,
+          DateFormat.MMMM_dd_yyyy,
+          DateFormat.EEEE_MM_dd_yyyy,
+          DateFormat.MMM_dd_EEEE,
+          DateFormat.EEEE_M_d,
+          DateFormat.EEE_MM_dd,
+        ].map((item) => (
+          <DropdownMenu.Item
+            key={item}
+            onSelect={() => {
+              if (item === selectedDateFormat) return;
+              setSelectedDateFormat(item);
+              setCanSave(true);
+            }}
+          >
+            <div className="h-10 bg-white color-gray-700 w-64 flex py-1 px-3 justify-between space-x-3 cursor-pointer items-center text-sm">
+              <div>{format(new Date(), item)}</div>
+              {item === selectedDateFormat && (
+                <HiCheckCircle className="h-4 w-4 text-primary" />
+              )}
+            </div>
+          </DropdownMenu.Item>
+        ))}
+      </DropdownMenu.Content>
+    </DropdownMenu.Root>
+  );
+
+  return (
+    <div className="flex space-x-6 justify-between">
+      {/* Form */}
+      <div className="mt-3">
+        <div className="font-medium">Language</div>
+        <div className="mt-3">{languageDropdown}</div>
+        <div className="font-medium mt-10">Intro sentence</div>
+        <div className="mt-3">{introSentenceElem}</div>
+        <div className="font-medium mt-10">Date format</div>
+        <div className="mt-3">{dateFormatDropdown}</div>
+        <div className="mt-10">
+          <button
+            className="button w-full"
+            disabled={!canSave || isUpdating}
+            onClick={() => {
+              updateFormatting({
+                dateFormat: selectedDateFormat,
+                introSentence,
+                introSentenceType,
+                language: selectedLanguage.key,
+              });
+            }}
+          >
+            Save
+          </button>
+        </div>
+      </div>
+
+      {/* Preview */}
+      <div className="relative flex">
+        <img src={previewImage} alt="preview" />
+        <div
+          className="absolute whitespace-pre"
+          style={{ top: "39%", left: "5%", fontSize: "0.9vw" }}
+          dangerouslySetInnerHTML={{
+            __html: extractTextFromSlots(
+              [
+                {
+                  start: addHours(startOfDay(new Date()), 10),
+                  end: addHours(startOfDay(new Date()), 11),
+                },
+                {
+                  start: addHours(startOfDay(new Date()), 15),
+                  end: addHours(startOfDay(new Date()), 16),
+                },
+              ],
+              {
+                formatLanguage: selectedLanguage.key,
+                dateFormat: selectedDateFormat,
+                copyTitle: introSentence,
+                highlight: {
+                  date: languageDropdownOpen || dateFormatDropdownOpen,
+                  introSentence: languageDropdownOpen || introSentenceFocused,
+                  time: languageDropdownOpen,
+                },
```

Comment:
what is actually highlighting here? i didn't see any difference in the ui

Reply:
Mentioned [here](https://github.com/agreeto/agreeto/pull/15#discussion_r1036764201)